### DISCUSSION TITLE
0.7.5

### DISCUSSION THREAD

**Discussion by simprecicchiani** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#135] [General]
⚠️ Not tested

- Opencore and Kexts updated to [November 2021 Release](https://dortania.github.io/hackintosh/updates/2021/11/01/acidanthera-november.html)

<hr /><em>This discussion was created from the release <a href='https://github.com/simprecicchiani/ThinkPad-T460s-macOS-OpenCore/releases/tag/0.7.5'>0.7.5</a>.</em>

**Comment by simprecicchiani** [comment]
For those who didn't know, unfortunately all the maintainers don't own a T460s anymore.  
Since I didn't find any significant changes in OC I wanted to provide you with an update, please report any unexpected problem or behaviour. Happy hackintoshing everybody.

**Reply by 559911** [reply]
It works for me.thx

**Reply by D35P4C1T0** [reply]
I'm using the intel wlan extra kexts for Big Sur and it looks like they don't work. Wifi can't be turned on at all, and I double checked everything, but I'm pretty sure I'm still missing something.

**Reply by alexdelli** [reply]
> For those who didn't knew, unfortunately all the maintainers don't own a T460s anymore.  
> Since I didn't find any significant changes in OC I wanted to provide you with an update, please report any unexpected problem or behaviour. Happy hackintoshing everybody.

You are the best! Thank you

**Reply by robertojguerra** [reply]
I will test with a yoga 260+intelairport+Monterey :) so excited!

**Reply by simprecicchiani** [reply]
> I'm using the intel wlan extra kexts for Big Sur and it looks like they don't work. Wifi can't be turned on at all, and I double checked everything, but I'm pretty sure I'm still missing something.

@D35P4C1T0 I may have changed the kext name from `AirportItlwm-11.kext` to `AirportItlwm-BigSur.kext`, sorry for the inconvenience. Please check if your filename match in `config.plist`.

**Reply by robertojguerra** [reply]
> I will test with a yoga 260+intelairport+Monterey :) so excited!

So I just tested it, with the required changes for my Yoga 260: adding the appropriate intel wifi plist children, my serial, uuid, and alcid=15... and it works perfectly :)
Thank you for updating the repository!

**Comment by nguyenthienhoan** [comment]
i have prolblem when boot to EFI:

OC: Grabbed zero system-id for SB, this is not allowed
Halting on critical error

**Reply by nguyenthienhoan** [reply]
oh, i was fix with edit string SecureBootModel - Default to Disabled

**Comment by 559911** [comment]
![image](https://user-images.githubusercontent.com/78580537/143169710-f44cc759-02c1-49cf-aeb8-b9b26a247893.png)
I find this problem.

**Reply by 559911** [reply]
![image](https://user-images.githubusercontent.com/78580537/143169820-231c3b89-8dab-4eb4-bbdf-0c7c5f43cf1b.png)
 I don't know how to fix it.
